brave-unbreak.txt

```diff
@@ -88,3 +87,27 @@
 ||pdfjs.robwu.nl
 ! Allow 1P piwik since it's no different than server logs
 @@https://analytics.*/piwik.$~third-party
+! Fully block Facebook everywhere just like our Disconnect blocking but unbreak logins like from quora.com and twitch.tv
+! Note that options will be added to exclude these filters soon.
+||facebook.com$third-party
+||facebook.net$third-party
+||staticxx.facebook.com$third-party
+@@||connect.facebook.com/*/sdk.js$script
+@@||connect.facebook.net/*/sdk.js$script
+@@||facebook.com/connect/
+@@||www.facebook.com/connect
+@@||staticxx.facebook.com/connect/
+@@||graph.facebook.com/
+! Block fbcdn.net everywhere but allow Facebook embeds
+||fbcdn.net$third-party,domain=~facebook.com
+@@||staticxx.facebook.com/
+@@||xx.fbcdn.net/
+@@||www.facebook.com/plugins/post.php?href=
```

Comment:
There are [a few more URLs](https://github.com/brave/brave-browser/issues/3241#issuecomment-465825247) for Facebook embeds:

- `www.facebook.com/plugins/video.php`
- `www.facebook.com/plugins/comment_embed.php`
- `www.facebook.com/plugins/page.php`

If you want to test them all at once, you an use the [test page I created last week](https://fmarier.github.io/brave-testing/social-widgets.html).